marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -294,68 +323,6 @@ handleAction _ (HandleWalletMessage Wallet.SendContractToWallet) = do
   contract <- toSimulation $ Simulation.getCurrentContract
   void $ query _walletSlot unit (Wallet.LoadContract contract unit)
 
-handleAction _ (JSHandleEditorMessage (Monaco.TextChanged text)) = do
-  liftEffect $ LocalStorage.setItem jsBufferLocalStorageKey text
-  assign _jsCompilationResult JSNotCompiled
-  assign _activeJSDemo ""
-
-handleAction _ (JSSelectEditorKeyBindings bindings) = do
-  assign _jsEditorKeybindings bindings
-  void $ query _jsEditorSlot unit (Monaco.SetKeyBindings bindings unit)
-
-handleAction _ CompileJSProgram = do
-  maybeModel <- query _jsEditorSlot unit (Monaco.GetModel identity)
-  case maybeModel of
-    Nothing -> pure unit
-    Just model -> do
-      assign _jsCompilationResult JSCompiling
-      maybeMarkers <- query _jsEditorSlot unit (Monaco.GetModelMarkers identity)
-      void $ subscribe
-        $ affEventSource
-            ( \emitter -> do
-                case map ((filter (\e -> isError e.severity)) <<< toUnfoldable) maybeMarkers of
-                  Just ({ message, startLineNumber, startColumn } : _) -> do
-                    emit emitter
-                      ( CompiledJSProgram $ Left
-                          $ JSI.CompilationError
-                              { row: startLineNumber
-                              , column: startColumn
-                              , text: [ message ]
-                              }
-                      )
-                    pure mempty
-                  _ -> do
-                    res <- JSI.eval model
-                    emit emitter (CompiledJSProgram res)
-                    pure mempty
-            )
```

Comment:
OOI, where did this go?

Reply:
a listener wasn't actually needed, it went to https://github.com/input-output-hk/plutus/blob/6783ed4003864adbd77c29d2317bd221d80a2c0a/marlowe-playground-client/src/JavascriptEditor/State.purs#L46